### DISCUSSION TITLE
Querying records with relationships in 0.23.12 - cant get it working

### DISCUSSION THREAD

**Discussion by designlook** [pocketbase/pocketbase#6196] [Q&A] [answered]
```
var sourceFiles []struct {
	Id   string `db:"id"`
	File string `db:"file"`
}

filesQuery := app.DB().Select("id", "file").
	From(filesTableName).
	Where(dbx.HashExp{"image_doc": imageDocId})

err = filesQuery.All(&sourceFiles)
```

The above fails when I try to query the filesTableName collection using image_doc which is a multiple relationship field. imageDocId is a string and id of the record in image_docs collection.

**Comment by ganigeorgiev** [comment, marked as answer]
Multi-valued fields are stored as serialized json array and if you want to perform plain SQL queries you'll have to use the [SQLite json functions](https://www.sqlite.org/json1.html) (in this particular case [`json_each`](https://www.sqlite.org/json1.html#the_json_each_and_json_tree_table_valued_functions)). For example:

```go
filesQuery := app.DB().Select("id", "file").
	From(filesTableName).
	LeftJoin("json_each(image_doc) as je", nil).
	Where(dbx.HashExp{"je.value": imageDocId})
```

**Reply by designlook** [reply]
Thanks! that worked! this is worth having in the docs. Thanks
